addons/launcher-addon/src/main/java/io/fabric8/launcher/addon/ui/booster/ProjectInfoStep.java

```diff
@@ -293,7 +293,10 @@ public Result execute(UIExecutionContext context) throws Exception {
             }
         } catch (Exception e) {
             if (e instanceof FileNotFoundException) {
-                logger.log(Level.WARNING, "No README.adoc template found for " + mission.getId());
+                logger.log(Level.WARNING, "No README.adoc and properties found for " + mission.getId() + " " + runtime.getId() + 
```

Comment:
No big deal, but it would be interesting to have two catch clauses instead and log appropriately

Reply:
You can also log e.getMessage() to know the file name that was not found